gallery.html

```diff
@@ -39,12 +39,10 @@
           </ul>
         </nav>
       </header>
-
-      <h2>Gallery</h2>
-      <br /><br />
       <main>
+        <h1>Gallery</h1>
         <div class="gallery">
-          <a row-span="2" href="https://loremflickr.com/1024/1024/?lock=50" >
+          <a class="row-span-2" href="https://loremflickr.com/1024/1024/?lock=50" >
```

Comment:
`row-span` is not an attribute. You were to give the element a class named `row-span-2`